### PR TITLE
README update with new cmdline arguments

### PR DESCRIPTION
**Pull request by crepricg**
Updates the launch instructions in the README.md file to include the cmd line arguments that are now needed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,15 +37,33 @@ JAVA&nbsp;7 when working with this project.
...
        ```
 
-     The `run_server` and `run_client` scripts have hard-coded addresses for
-     your local machine. If you are running the server on a different machine
-     than the client, you will need to change the host portion of the address
-     in `run_client.sh` to the name of the host where your server is running.
-     Make sure the client and server are using the same port number.
+     You must specify the following startup arguments for `run_server.sh:
+     + `<team_id>` and `<team_secret>`: a numeric id for your team, and a secret
+       code, which are used to authenticate your server with the Relay server.
```
Comment (vaage) [inline README.md]: The secrete must be HEX (ie. a string of a-fA-F0-9)

Comment (crepricg) [inline README.md]: The server takes any  string as input, and unless one connects to the Relay the secret is never used. Plus Secret.parse() can handle non-hex characters, so any char out of range will be seen as 0, but I'll add the specs anyway :)

Comment (vaage) [inline README.md]: Thanks. Secret.parse really should be changed to throw an IOException.

File: README.md [hunk 2]
```diff
@@ -44,9 +44,10 @@ JAVA&nbsp;7 when working with this project.
      You must specify the following startup arguments for `run_server.sh:
      + `<team_id>` and `<team_secret>`: a numeric id for your team, and a secret
        code, which are used to authenticate your server with the Relay server.
-       You can specify any integer value for `<team_id>`, and any string for
-       `<team_secret>` when you launch the server in your local setup since it
-       will not connect to the Relay server.
+       You can specify any integer value for `<team_id>`, and a value expressed
+       in hexadecimal format (using numbers `0-9` or letters in the range `A-F`)
```
Comment (vaage) [inline README.md]: NIT: Should this be 'or' or 'and'?

Comment (crepricg) [inline README.md]: I think both  work. I changed it to and.

**Pull request merged**